daemon/oci_linux.go

```diff
@@ -661,7 +662,6 @@ func (daemon *Daemon) createSpec(c *container.Container) (*libcontainerd.Spec, e
 		return nil, err
 	}
 	ms = append(ms, c.IpcMounts()...)
-	ms = append(ms, c.TmpfsMounts()...)
```

Comment:
Looks like `TmpfsMounts` is now dead code and can be removed with this change (and is in several platform files).